### PR TITLE
Check out Ansible repo

### PR DESCRIPTION
**Pull request by mkrizek**
Sorry for some unrelated changes but those were needed to run the script (syntax errors, ...).

### CODE REVIEW

**Comment by mkrizek** [comment]
cc @bcoca @webknjaz

File: migrate.py [hunk 1]
```diff
@@ -29,6 +30,53 @@
...
+        'so': so,
+        'se': se
+    }
+
+
+def checkout_repo(refresh=False):
+    releases_dir = os.path.join(VARDIR, 'releases')
+    devel_path = os.path.join(releases_dir, 'devel.git')
+
+    if refresh and os.path.exists(devel_path):
```
Comment (webknjaz) [inline migrate.py:67]: If we aren't limited by outdates Pythons I'd prefer pathlib to be used instead.

File: migrate.py [hunk 2]
```diff
@@ -29,6 +30,53 @@
...
+        # TODO do we want/is it worth to use a git library instead?
```
Comment (webknjaz) [inline migrate.py:68]: I contributed to `pygit2` a while back so they ship manylinux1 wheels now. There's also `GitPython` but I haven't tried this one.

File: migrate.py [hunk 3]
```diff
@@ -29,6 +30,53 @@
 PLUGIN_EXCEPTION_PATHS = {'modules': 'lib/ansible/modules', 'module_utils': 'lib/ansible/module_utils'}
 
 
+def _run_command(cmd=None, check_rc=True):
+    logger.debug(cmd)
+    if not isinstance(cmd, bytes):
+        cmd = cmd.encode('utf-8')
+    p = subprocess.Popen(
```
Comment (webknjaz) [inline migrate.py:37]: Maybe just check_run/call?

Comment (mkrizek) [reply migrate.py:37]: I didn't want to change run_command functions much in this PR. I'd prefer to do that separately.

File: migrate.py (hunk 1 above)
Comment (bcoca) [reply migrate.py:67]: not what we should be focusing on, if the current functions can work with minor fix, i prefer to do that, we can change it later on once all the actual features are working

File: migrate.py (hunk 2 above)
Comment (bcoca) [reply migrate.py:68]: same as above

File: migrate.py (hunk 3 above)
Comment (bcoca) [reply migrate.py:37]: agreed, lets concentrate on functional changes needed to get this working

Comment (webknjaz) [reply migrate.py:37]: Ah, I initially thought that you're adding this function, not moving it.

**Pull request merged**